app/pages/admin/project-status.cjsx

```diff
@@ -92,6 +95,43 @@ ProjectExperimentalFeatures = React.createClass
   setting: (task) ->
     task in (@props.project?.experimental_tools or [])
 
+  handleDisciplineTagChange: (value) ->
+    newTags = if value is '' then [] else value.split(',')
+    @setState disciplineTagList: newTags
+    allTags = newTags.concat @state.otherTagList
+    @handleTagChange(allTags)
+
+  handleOtherTagChange: (value) ->
+    newTags = if value is '' then [] else value.split(',')
+    @setState otherTagList: newTags
+    allTags = @state.disciplineTagList.concat newTags
+    @handleTagChange(allTags)
+
+  handleTagChange: (value) ->
+    event =
+      target:
+        value: value
+        name: 'tags'
+        dataset: {}
+    handleInputChange.call @props.project, event
+
+  splitTags: (kind) ->
+    disciplineTagList = []
+    otherTagList = []
+    for t in @props.project.tags
+      if DISCIPLINES.some((el) -> el.value == t)
```

Comment:
yes, it is the same as https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/lab/project-details.cjsx. I am not actually sure how to do react mixins but will take a look if it can be done easily.